ci_watson/plugin.py

```diff
@@ -81,3 +83,50 @@ def _jail(tmp_path):
 def envopt(request):
     """Get the ``--env`` command-line option specifying test environment"""
     return request.config.getoption("env")
+
+
+@pytest.fixture(scope="module")
+def resource_tracker():
+    """Fixture to return the current module-scoped ResourceTracker.
+
+    Use by calling ``track`` to generate a context in which resource
+    usage will be tracked.
+
+    >>>
+    >> with resource_tracker.track():
+    >>     # do stuff
```

Comment:
Why `>>` instead of `>>>`?

```suggestion
    >>> with resource_tracker.track():
    >>>     # do stuff
```

Reply:
These aren't doctests but by using `>>` they will get formatted/syntax highlighted like they are.

Reply:
Maybe what you need instead is this:

```rst
.. code-block:: python

    with resource_tracker.track():
        # do stuff
```

This would format the code and skip doctest.

Reply:
Thanks. That doesn't highlight the code in vim (I'm not sure about other editors). I liked that it highlighted the code snippets for me but if it's preferred to use the sphinx directive in the docstring I'll make the changes.

Reply:
I switched these all to `code-block` in https://github.com/spacetelescope/ci_watson/pull/74/commits/054bdf5a0296cbfaf4829a3ea71b3b24958271fb Let me know if I missed any.

Reply:
Just that `>>` is a non-standard usage AFAIK. Thanks for your understanding!